themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: GCP-Native
+meta_desc: This page provides on an overview of the Native Google Cloud (GCP-Native) provider for Pulumi.
+menu:
+  intro:
+    parent: cloud-providers
+    identifier: clouds-gcp-native
+    weight: 1
+
+aliases: ["/docs/reference/clouds/gcp-native/"]
+---
+<div class="note note-info" role="alert">
+    <p>
+        GCP-Native provider is currently in <strong> public preview </strong>
+    </p>
+</div>
+
+
+<img src="/logos/tech/gcp.svg" align="right" class="h-16 px-8 pb-4">
+
+The Google Cloud Native (GCP-Native) provider for Pulumi can be used to provision any of the cloud resources available in [GCP](https://cloud.google.com/).  The GCP provider must be configured with credentials to deploy and update resources in Google Cloud.
+
+See the [full API documentation]({{< relref "/docs/reference/pkg/gcp-native" >}}) for complete details of the available GCP provider APIs.
+
+## Setup
+
+The GCP provider supports several options for providing access to Google Cloud credentials.  See [GCP setup page]({{< relref "setup" >}}) for details.
+
+## Getting Started
+
+GCP Native is currently in *public preview*. The quickest way to get started with GCP is to follow the steps described in the [README](https://github.com/pulumi/pulumi-gcp-native#readme).
+
+Some interesting examples are available complete with instructions:
+
+* [Google Cloud Functions](https://github.com/pulumi/examples/tree/master/gcp-ts-functions): Create a serverless function
```

Comment:
TODO - point to GCP native versions of these.

Reply:
This one can be reused from the gcp-native repo

Reply:
> This one can be reused from the gcp-native repo

See https://github.com/pulumi/examples/pull/967